infra/modules/aws/main.tf

```diff
@@ -120,6 +120,12 @@ resource "aws_ssm_parameter" "salt" {
   type        = "SecureString"
   description = "Salt used to build pseudonyms"
   value       = sensitive(random_password.random.result)
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
```

Comment:
OK, although a bit weird.  should certainly comment this. I think it's entirely legitimate for us to pre-generate a value, and then they overwrite it if they don't trust ours - and if so, our terraform doesn't replace it - but still should comment that that's what's happening.